src/include/MaxxEffect/Control/Direct/MaxxEffect_Revision.h

```diff
@@ -0,0 +1,32 @@
+/*******************************************************************************
+ * Library revision identification
+ ******************************************************************************/
+#ifndef MAXX_EFFECT_REVISION_H
+#define MAXX_EFFECT_REVISION_H
+
+#include "MaxxEffect/MaxxEffect.h"
```

Comment:
All these headers should be under ```src/include/sof/audio/```

Reply:
We als need a SPDX licence at the top of each file too.

Reply:
@lgirdwood regarding files placement
our API uses relative include paths like `#include "MaxxEffect/MaxxEffect.h"`
this requires that a folder with API to be on includes path
by default `src/include/sof/audio/` is not on path
will it be ok if I add it on path? something like this:
```
--- a/src/audio/codec_adapter/CMakeLists.txt
+++ b/src/audio/codec_adapter/CMakeLists.txt
@@ -13,4 +13,5 @@ endif()
 if(CONFIG_WAVES_CODEC)
 add_local_sources(sof codec/waves.c)
 sof_add_static_library(MaxxChrome ${CMAKE_CURRENT_LIST_DIR}/lib/release/libMaxxChrome.a)
+target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/)
 endif()
```

Reply:
@lgirdwood moved files and added SPDX licence identifier

Reply:
Yep, adding this to Makefile is good.